zimports.py

```diff
@@ -182,20 +185,20 @@ def _write_source(
                 ):
                     buf.append("")
                 previous_import = import_node
-                buf.append(_write_import(import_node))
+                buf.extend(_write_import(import_node, max_line_length=max_line_length))
 
             for import_node in nosort_imports:
                 if previous_import is not None:
                     buf.append("")
                     previous_import = None
-                buf.append(_write_import(import_node))
+                buf.extend(_write_import(import_node, max_line_length=max_line_length))
 
         if lineno not in import_gap_lines:
             buf.append(line.rstrip())
     return buf
 
 
-def _write_import(import_node):
+def _write_import(import_node, max_line_length=200):
```

Comment:
can this be a positional argument without the default?  there are two other default max line lengths hardcoded.